src/main/java/devcource/hihi/boardjpa/domain/Post.java

```diff
@@ -0,0 +1,66 @@
+package devcource.hihi.boardjpa.domain;
+
+import devcource.hihi.boardjpa.dto.post.CreatePostDto;
+import devcource.hihi.boardjpa.dto.post.ResponsePostDto;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import java.util.Objects;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Post extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "post_id")
+    private Long id;
+
+    @NotNull
+    private String title;
+
+    private String content;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @Builder
+    public Post(String title, String content) {
+        this.title = title;
+        this.content = content;
+    }
+
+    @Builder
+    public Post(String title, String content,User user) {
+        this.title = title;
+        this.content = content;
+        this.user = user;
+    }
```

Comment:
```suggestion
    @Builder
    public Post(String title, String content) {
        this(title, content, null)
    }

    @Builder
    public Post(String title, String content,User user) {
        this.title = title;
        this.content = content;
        this.user = user;
    }
```

p1; 
값에 대한 검증이 필요해보여요! 만약 title, content 길이가 255를 넘어간다면 어떻게 될까요?